hw/dv/tools/dvsim/common_sim_cfg.hjson

```diff
@@ -84,30 +84,25 @@
   ]
 
   // Build modes are collection of build_opts and run_opts
-  // A test can enable a specific build mode by setting 'use_build_mode' key
```

Comment:
IMO this comment is useful.

Reply:
I agree. A rephrased version of that sentence is at the bottom of the comment block.